common/components/card/template.njk

```diff
@@ -63,6 +63,16 @@
       </ul>
     {% endif %}
 
+    {% if params.importantEventsTagList.length %}
```

Comment:
Can we not re-use the existing functionality that this card already has to display tags.

These are being rendered using the same markup so I think it would be better to re-use them. There is already a way to visually distinguish them so we can do without this.